src/software/SfM/export/main_openMVG2openMVS.cpp

```diff
@@ -86,6 +86,11 @@ bool exportToOpenMVS(
   scene.images.reserve(nViews);
   for (const auto& view : sfm_data.GetViews())
   {
+    if (map_intrinsic.count(view.second->id_intrinsic) == 0) 
```

Comment:
I would recommend to use `IsPoseAndIntrinsicDefined` : https://github.com/openMVG/openMVG/blob/develop/src/openMVG/sfm/sfm_data.hpp#L61

Reply:
Thanks for your advice. 

Fixed, and removed some redundant and ambiguous codes (in my opinion).